haystack/components/generators/chat/openai.py

```diff
@@ -19,11 +19,11 @@
 @component
 class OpenAIChatGenerator:
     """
-    Enables text generation using OpenAI's large language models (LLMs). It supports gpt-4 and gpt-3.5-turbo
+    Enables text generation using OpenAI's large language models (LLMs). It supports `gpt-4` and `gpt-3.5-turbo`
     family of models accessed through the chat completions API endpoint.
 
     Users can pass any text generation parameters valid for the `openai.ChatCompletion.create` method
-    directly to this component via the `**generation_kwargs` parameter in __init__ or the `**generation_kwargs`
+    directly to this component via the `**generation_kwargs` parameter in `__init__` or the `**generation_kwargs`
```

Comment:
```suggestion
    directly to this component via the `generation_kwargs` parameter in `__init__` or the `generation_kwargs`
```

Formerly, these were keyword arguments. Now they are dicts...

Please also apply this change in  `AzureOpenAIChatGenerator`.

Reply:
👍🏽